rpc/block_svc_routines.c

```diff
@@ -4268,9 +4270,10 @@ block_create_common(blockCreate *blk, char *rbsize, char *prio_path)
 
 
   LOG("mgmt", GB_LOG_INFO,
-      "create request, volume=%s blockname=%s blockhosts=%s filename=%s authmode=%d "
-      "passwd=%s size=%lu", blk->volume, blk->block_name, blk->block_hosts,
-      blk->gbid, blk->auth_mode, blk->auth_mode?blk->passwd:"", blk->size);
+      "create request, volume=%s volserver=%s blockname=%s blockhosts=%s "
+      "filename=%s authmode=%d passwd=%s size=%lu", blk->volume, blk->block_name,
+      volServer?volServer:blk->ipaddr, blk->block_hosts, blk->gbid, blk->auth_mode,
+      blk->auth_mode?blk->passwd:"", blk->size);
```

Comment:
The blockname and volserver are reversed here:

`2068 [2018-11-15 08:17:53.521108] INFO: create request, volume=dht volserver=block3 blockname=192.168.195.162 blockhosts=192.168.195.162,192.168.195.163,192.168.195.164 file     name=1ef85fcf-cd36-480d-b39f-1a887b59796e authmode=0 passwd= size=1073741824 [at block_svc_routines.c+4282 :<block_create_common>]`

Reply:
Good catch @lxbsz, fixed now. Thanks!